### DISCUSSION TITLE
Add URL queries to Energy dashboard

### DISCUSSION THREAD

**Discussion by DarkC35** [home-assistant/frontend#13880] [Dashboard]
Hi it would be nice to have URL query support for the energy dashboard. Currently this dashboard always has the `/energy` domain and it opens either "today" or the last (cached?) opened configuration.

This would be great for Hyperlinks on other dashboards (e.g. "Consumption yesterday", "Consumption last week", ...) or actionable notifications (in my case I automatically import historic data since my energy provider only provides a daily report of the past day, therefore a notification like "New energy data imported (higher than usual)..." -> "Show inserted data" should open "yesterday" and not "today").

Example:
`/energy?start_date=2022-09-25&period=day` where `start_date` is the first day in the "period" and `period` is one of the supported groupings/cumulations (day, week, month, year).

Edit:
I looked into the frontend code and used the available variable names for the selection in snake case to match the usage of the `/history` page.

**Comment by karolwolek** [comment]
I would really love this feature too!

**Comment by matt-horwood-mayden** [comment]
I have found this when looking to use TRMNL HA with the energy dashboard to send a screenshot to a slack channel, but I want last weeks energy not now.

How would I do that?
